test/Shared.ps1

```diff
@@ -1,20 +1,44 @@
 $modulePath = Convert-Path $PSScriptRoot\..\src
 $moduleManifestPath = "$modulePath\posh-git.psd1"
 
+$csi = [char]0x1b + "["
+if (($PSVersionTable.PSVersion.Major -le 5) -or $IsWindows) {
+    # On Windows, we can access the git binary via git.exe
+    $global:gitbin = Get-Command -Name git -CommandType Application
+}
+else {
+    # On Linux/macOS, we can access the git binary via its path /usr/bin/git
+    $global:gitbin = (Get-Command -Name git -CommandType Application).Path
```

Comment:
I've got two versions of git installed on Mac OS, and iirc that's normal with Homebrew, so test using this break at the moment. `(Get-Command blah)[0]` fixes them.

Reply:
Added `-TotalCount 1`